db/schema.rb

```diff
@@ -11,7 +11,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170808162101) do
+
+
+
+ActiveRecord::Schema.define(version: 20170809153751) do
```

Comment:
Looks like you need to re-dump the schema here. password/share_type

Reply:
done